web-src/src/webapi/index.js

```diff
@@ -267,6 +267,21 @@ export default {
     })
   },
 
+  library_composers () {
+    return axios.get('/api/library/composers')
+  },
+
+  library_composer (composer, type) {
+    var params = {
+      type: type,
+      media_kind: 'music',
+      expression: 'composer is "' + composer + '"'
+    }
+    return axios.get('/api/search', {
+      params: params
+    })
+  },
```

Comment:
To keep it consistent with the genre functions, please split this into two functions (`library_composer` and `library_compose_tracks`)

Reply:
it removes duplication of code but made the change as per request